### PR TITLE
Topic/fix clang

### PR DESCRIPTION
**Pull request by hzhuang1**

### CODE REVIEW

**Comment by antonio-nino-diaz-arm** [comment]
IP Review +1

File: plat/hisilicon/hikey/include/plat_macros.S [hunk 1]
```diff
@@ -16,7 +16,7 @@
 gicc_regs:
 	.asciz "gicc_hppir", "gicc_ahppir", "gicc_ctlr", ""
 gicd_pend_reg:
-	.asciz "gicd_ispendr regs (Offsets 0x200 - 0x278)\n"	\
+	.asciz "gicd_ispendr regs (Offsets 0x200 - 0x278)\n",	\
```
Comment (antonio-nino-diaz-arm) [inline plat/hisilicon/hikey/include/plat_macros.S]: Doesn't this split the string into two strings? Doesn't the comma add a '\0'?

**Comment by hzhuang1** [comment]
@antonio-nino-diaz-arm Merge the two lines into one.

**Comment by soby-mathew** [comment]
@hzhuang1 , we will merge this after v2.0 release is done which is expected to be next week.

**Comment by hzhuang1** [comment]
It's fine to me.

**Pull request merged**